README.md

```diff
@@ -1,2 +1,52 @@
 # ft_transcendence
-🏓 Multiplayer pong tournaments
+🏓 Play pong online with your friends in real time
+
+# 🤝 Contributing
+
+If you want to contribute to this project you may follow a set of rules.
+
+### 1. Create a new issue or task:
+
+- With every contribution there must be a related issue/task, so if you
+don't have any, create a new one on the project board with some
+detailed info about what do you want to accomplish.
+
+### 2. Create a new branch:
+   
+- ⚠️ We **do not** push to `master`, never. So you'll need to create a branch
```

Comment:
suggestion: What do we think about replacing `master` with `main`? To reflect the branch naming we have.